src/cartridges/bm_adyen/cartridge/static/default/js/adyenSettings.js

```diff
@@ -465,6 +468,14 @@ document.addEventListener('DOMContentLoaded', () => {
     }
   });
 
+  productionRadio.addEventListener('click', () => {
+    livePrefix.disabled = false;
+  });
+
+  testRadio.addEventListener('click', () => {
+    livePrefix.disabled = testRadio.value === 'TEST';
```

Comment:
this should always be `true`
```
livePrefix.disabled = true;
```

Reply:
testRadio.value will always be TEST, however changed it to be true.